pandas/core/generic.py

```diff
@@ -11221,6 +11221,34 @@ def _doc_params(cls):
 age       16.269219
 height     0.205609"""
 
+_var_examples = """
+
+Examples
+--------
+>>> df = pd.DataFrame({'person_id': [0, 1, 2, 3],
+...                   'age': [21, 25, 62, 43],
+...                   'height': [1.61, 1.87, 1.49, 2.01]}
+...                  ).set_index('person_id')
+>>> df
+           age  height
+person_id
+0           21    1.61
+1           25    1.87
+2           62    1.49
+3           43    2.01
+
+The variance of the columns can be found as follows:
```

Comment:
Nit: I think this sentence can be removed (somewhat obvious from the beginning of the doc string)